nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/main/java/org/nuxeo/ecm/platform/rendering/template/DocumentModelAdapter.java

```diff
@@ -74,11 +76,24 @@ public TemplateModel get(String key) throws TemplateModelException {
         // may be a schema name (doc.dublincore.title)
         Map<String, Object> properties = doc.getProperties(key);
         if (properties != null) {
-            return wrapper.wrap(properties);
+            return wrapper.wrap(unPrefixedMap(properties));
         }
         return wrapper.wrap(null);
     }
 
+    private static Map<String, Object> unPrefixedMap(Map<String, Object> props) {
+        Map<String, Object> res = new HashMap<String, Object>();
```

Comment:
new HashMap<String, Object>() can be replaced by new HashMap<>() the types are not required.

Reply:
good to know, thanks